src/PluginsSystem/Tools/Microsoft.Performance.Toolkit.Plugins.Cli/Utils.cs

```diff
@@ -0,0 +1,23 @@
+﻿
+namespace Microsoft.Performance.Toolkit.Plugins.Cli
+{
+    public static class Utils
+    {
+        public static string GetValidDestFileName(string file)
```

Comment:
Nit: the name of this method doesn't explain what it does. What does "Valid" mean in this context?